### DISCUSSION TITLE
Setting Up Munify

### DISCUSSION THREAD

**Discussion by ady00** [DeutscheModelUnitedNations/munify#81] [Support]
Hey there!
I recently found Munify, and would love to use it for an upcoming conference. 
Currently, I've got the chase front end and backend working for the most part locally, but I can't  seem to log in as a chair (e.g I can't make any changes to the chair side of the app) on my local build. Is there a setup guide available for Munify?

Thanks so much!

**Comment by m1212e** [comment]
Hi, first of all: Thanks for your interest in munify! Currently we are still in active development. Many parts of the app aren't implemented yet and will be added in the future. If you would like to contribute, please see the contribution section in the README.

I will add a disclaimer to the README and also open an issue for creating a setup guide in how to self deploy and use the app once it's finished.

**Reply by ady00** [reply]
Sure, sounds good! I'd love to help contribute.
Currently, how can I access the chair backend while running locally?

Thanks so much!

**Reply by m1212e** [reply]
I'll assume some knowledge about the whole JS/Node/Bun ecosystem, if you don't understand something feel free to ask! One of my TODOs is refreshing the getting started with development doc of the project, so currently there may be not much information on how you can get the whole thing up and running.

First, you need to install all deps and the bun runtime. Then use the package json scripts in the front and backend to start the dev servers. We use a typesafe API framework called "Elysia.js" which allows you to call strongly typed endpoints. Best would be to take a look at the frontends contexts which allow you to connect and call the backend!

**Reply by m1212e** [reply]
Also, if you are not so sure where to get started, you may consider taking a look at the open issues and work on one of those.
If your question is answered, please mark it accordingly!
Thanks again for your interest in the project!

**Reply by ady00** [reply]
Yup, I already have the server and webapp running.
How do I utilize the mockedAuth settings in the backend? Currently, I've been changing the values of mockedUser and mockedPermission to no avail.

**Reply by m1212e** [reply]
Best would probably be to ignore, comment out or remove auth from endpoints you would like to use for now. Auth is a topic which I currently work on and which is going to change a lot. 
Since @TadeSF  and I are currently the most active, if you'd really like to dig into it we might just hop on a call together, explain our plans a little and show you where we want to head with the project?
